### PR TITLE
Conv VAE notebook tutorial

### PR DESCRIPTION
**Pull request by Bob-Chou**
Migrated Conv VAE notebook tutorial

### CODE REVIEW

**Comment by yangw1234** [comment]
seems the images does not change after the 1st epoch
looks like a bug in KerasModel

**Comment by Bob-Chou** [comment]
I guess it's probably caused by the convergence process. 

After the first epoch, the model has nearly reached the optimum, and so the later epochs could only introduce slight changes. 

It's hard to be recognized by comparing static images. But if we take a look at the last GIF (generated by shown output images), we could tell the evolution of generated images in 10 epochs.

**Comment by yangw1234** [comment]
can you try on this https://github.com/intel-analytics/analytics-zoo/pull/1469

**Comment by Bob-Chou** [comment]
Entered an error. It might be caused by a typo: there was no parentheses at the end of line 144 of `analytics-zoo/pyzoo/zoo/tfpark/model.py`.
```
TypeError                                 Traceback (most recent call last)
<ipython-input-11-0537ca55fbbc> in <module>
      3     vae.model.fit(train_dataset,
      4                   epochs=1,
----> 5                   distributed=True)
      6     generate_and_save_images(vae, epoch, random_vector_for_generation)

~/00_Workspace/00_Zoo/my-analytics-zoo/pyzoo/zoo/tfpark/model.py in fit(self, x, y, batch_size, epochs, validation_split, validation_data, distributed, **kwargs)
    124             if isinstance(x, TFNdarrayDataset):
    125                 x = _standarize_feature_label_dataset(x, self.model)
--> 126             self._fit_distributed(x, validation_split, epochs, **kwargs)
    127 
    128         elif distributed:

~/00_Workspace/00_Zoo/my-analytics-zoo/pyzoo/zoo/tfpark/model.py in _fit_distributed(self, dataset, validation_split, epochs, **kwargs)
    142     def _fit_distributed(self, dataset, validation_split, epochs, **kwargs):
    143         self.tf_optimizer = TFOptimizer.from_keras(self.model, dataset,
--> 144                                                    val_spilt=validation_split, **kwargs)()
    145         self.tf_optimizer.optimize(MaxEpoch(epochs))
    146 

TypeError: 'TFOptimizer' object is not callable
```
I tried deleting the parentheses and it worked). The updated result has been committed.

File: tensorflow/notebooks/cvae.ipynb [hunk 1]
```diff
@@ -0,0 +1,1181 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ITZuApL56Mny"
+   },
+   "source": [
+    "![evolution of output during training](https://tensorflow.org/images/autoencoders/cvae.gif)\n",
```
Comment (yangw1234) [inline tensorflow/notebooks/cvae.ipynb]: the gif should be replaced to ours

File: tensorflow/notebooks/cvae.ipynb [hunk 2]
```diff
@@ -0,0 +1,1181 @@
...
+    "Compile the keras model and wrap it with `KerasModel` for distributed training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vae.model.compile(optimizer, loss=compute_loss)\n",
```
Comment (yangw1234) [inline tensorflow/notebooks/cvae.ipynb]: not sure `loss=compute` is correct. see the official Keras implementation https://keras.io/examples/variational_autoencoder/

They use the `add_loss` api

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**